### DISCUSSION TITLE
Contributions not showing up

### DISCUSSION THREAD

**Discussion by ghost** [community/community#43529] [New to GitHub] [answered]
Hello, 

My contributions are not showing up on Github. I see the files are uploaded onto Github after committing it and pushing it from my local computer. 

I don't see the green squares filled in.

**Comment by ghost** [comment, marked as answer]
I solved it. I was just typing in, git push, when it should be git push origin main. Contributions are now showing up.

**Reply by queenofcorgis** [reply]
Hey @kalialor - so glad to hear you figured this out 🚀 Thanks for coming back to share the solution with us!

**Reply by yahyaerdoan** [reply]
Hi, my  contributions are not showing up on Github. Can you help me

**Comment by tobias-barakaa** [comment]
I was facing the same error, so I did **git remote -v** to check whether am pushing to the wrong repository but that wasn't the issue so I opted to check my email using **git config user.email** and since I've got many emails turns out I used a different email while configuring my GitHub so I changed it using **git config user.email "Enter your email here"** then proceeded to add a fake file and pushed using  **git push --set-upstream origin master**, went to check my GitHub account and wow

**Comment by Alhrafas** [comment]
i was facing this same problem when i push my code i see the code are uploaded but contribution is not any square whats wrong??

**Comment by Alhrafas** [comment]
my  contribution squares does not show up when i committed and push some changes

**Comment by karimahcodes** [comment]
my January commits aren't showing up on the activity section of my profile. I meet all the requirements on the github docs page, I commit to main/default branch, own the repository, and the commits show up in the repository itself.

**Comment by yahyaerdoan** [comment]
my chart of contributions not working can you help me

**Comment by Shankyking25** [comment]
I am also facing the same issue

**Comment by Shaurya25071997** [comment]
My github is showing an error occured while seeing my repository

**Comment by donald-okara** [comment]
My contributions are not visible on my profile. I have checked and the commits get pushed t the repository by MY ACCOUNT (username and email match) and I push to the master branch. What is the cause of this?
